src/commands/math/basicSymbols.js

```diff
@@ -351,13 +351,14 @@ LatexCmds.Pi =
 LatexCmds.Sigma =
 LatexCmds.Phi =
 LatexCmds.Psi =
-LatexCmds.Omega =
-LatexCmds.forall = P(VanillaSymbol, function(_, super_) {
+LatexCmds.Omega = P(VanillaSymbol, function(_, super_) {
```

Comment:
Just to double check, I'm guessing we're leaving these Greek symbols alone because they aren't mentioned in the AC's for this ticket?

Reply:
Yeah, they could have been mentioned but they weren't. (`forall` was mentioned, so I had to take it out of this list to map it to ∀)